frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerLifeCycle.java

```diff
@@ -45,6 +49,35 @@ public WorkerLifeCycle(ConfigManager configManager, Model model) {
         this.model = model;
     }
 
+    private String[] getEnvString(String cwd, String modelPath) {
+        ArrayList<String> envList = new ArrayList<>();
+        String blackList = "";
+        try {
+            blackList = configManager.getBlacklistEnvVars();
+        } catch (PatternSyntaxException e) {
```

Comment:
If the regex is checked at startup time, we don't need catch this exception here.